universal-login-sdk/lib/api/sdk.ts

```diff
@@ -66,6 +66,11 @@ class UniversalLoginSDK {
     this.featureFlagsService = new FeatureFlagsService();
   }
 
+  async getNotice(): Promise<string> {
```

Comment:
Make it not async, just return value from config (can be null/empty) if not loaded.